cms/jinja2/templates/components/streamfield/topic_headline_figure_block.html

```diff
@@ -1,19 +1,22 @@
 {% from "components/card/_macro.njk" import onsCard %}
 
+{% set error_text = _("Error") %}
+{% set error_supporting_text = _("Error retrieving figure") %}
+
 {# fmt:off #}
     {{
         onsCard({
             "variant": 'feature',
             "title": {
                 "id": figure.figure_id + "-card",
                 "headingLevel": 3,
-                "text": figure.title,
-                "url": figure.url,
+                "text": figure.title if figure.title else error_text,
+                "url": figure.url if figure.url else "",
```

Comment:
NP:
```suggestion
                "text": figure.title or error_text,
                "url": figure.url or "",
```

Same for the two changes below